saleor/graphql/account/mutations/account.py

```diff
@@ -143,6 +145,10 @@ def save(cls, info, user, cleaned_input):
             )
         else:
             user.save()
+
+        assign_user_gift_cards(user)
```

Comment:
Maybe I am missing something but is it safe? 
If we have disabled `ENABLE_ACCOUNT_CONFIRMATION_BY_EMAIL`, I could create customers based on real email addresses and try to steal possible assigned giftCards or fetch customer's Order data

Reply:
Used gift cards are always connected with gift cards, so even if the card will be not assigned to the new user, only this user will be able to use the gift card with the user email. 
What about the order, I'm not sure if getting the order data is dangerous. 

@patrys What do you think?